mailpoet/tests/_support/AcceptanceTester.php

```diff
@@ -650,6 +670,19 @@ public function checkEmailWasReceived($subject) {
     }
   }
 
+  /**
+   * Checks that email was not received by looking for a subject in inbox.
+   * @param string $subject
+   */
+  public function checkEmailWasNotReceived($subject) {
+    $i = $this;
+    $i->amOnMailboxAppPage();
+    $i->dontSee($subject);
+    // click refresh button to seek for new emails once again
+    $i->click('.glyphicon-refresh');
+    $i->dontSee($subject);
+  }
```

Comment:
This won't be a very reliable check unless we make sure the cron was running, or add some kind of wait (easy, but slow). Email sending and delivery takes some time, and in most cases this will pass because the emails was not **yet** received. I think force-triggering Action Scheduler could work here. You could do something like this:

```php
    $i = $this;
    $i->triggerMailPoetActionScheduler();
    $i->amOnMailboxAppPage();
    $i->dontSee($subject);
```

Maybe in this case, the refresh isn't even needed? I let you choose...